### DISCUSSION TITLE
Notification delay error

### DISCUSSION THREAD

**Discussion by Onewpus** [tribalfs/GalaxyMaxHzPub#247] [Q&A]
After installing the beta version, I've been delayed in notification on my Zalo app for a long time

**Comment by tribalfs** [comment]
[FAQ #18](https://github.com/tribalfs/GalaxyMaxHzPub/wiki/Frequently-Asked-Questions-(FAQs)#18-what-is-doze-mode-and-how-does-quick-doze-mod-works)
